src/index.js

```diff
@@ -168,6 +169,8 @@ function createConfig(options, entry, format) {
 	let cjsMain = replaceName(pkg['cjs:main'] || 'x.js', mainNoExtension);
 	let umdMain = replaceName(pkg['umd:main'] || 'x.umd.js', mainNoExtension);
 
+	const useBabel = !!pkg.babel || await isFile(resolve(cwd, './.babelrc')) || await isFile(resolve(cwd, './.babelrc.js'));
```

Comment:
I'm thinking that, as long as we can get a default babel config that is as optimized as Bublé, we can just default to Babel.

Reply:
Good stuff happening in terms of babel@7 in terms of outputting better tree-shakeable code & more concise/performant code, but it's still a beta. Im pretty confident in it, we have a comprehensive test suite after all, so I'm using it all over the place - aint sure though if we should promote it just yet as part of other tool (`microbundle` in this context) as it has potential to break some stuff. And also it might not be yet compatible with some other tooling as not every tool had an opportunity to catch up yet and provide support for it.

If we are gonna default to babel@7, Im wondering how do you plan to support babel configs? Or maybe you'd like to ignore it all along and just use internal, optimized one? Im not sure how the whole thing applies to your 0 configuration goal 😉 

Personally I'd say it's better to read the real babel config, but then I'm not sure how we should "optimize" it. Mutating the input and i.e. parsing it to add `loose` option to plugins & presets seems a little bit magical. If one specify his/her config we should just use it as is, right?

Reply:
Idea - we could possibly support babel config files without any magic, read the config files (only for validation) with some utility exposed by babel like this [one](https://github.com/babel/babel/pull/7201) and parse it (ideally we could just receive the result from babel in JSON) to **warn** people against using non-loose / costly stuff. WDYT?

Reply:
@Andarist I spoke to Henry and I believe he mentioned they pulled the configuration parsing & lookup out of Babel into a standalone module. Maybe we can use that now?

+1 for warning, though it'd be nice to have a way to use a "stock" babel config and let microbundle tweak it into loose mode.